cardano-crypto-tests/src/Test/Crypto/DSIGN.hs

```diff
@@ -127,6 +132,9 @@ defaultSigGen = do
   msg :: Message <- arbitrary
   signDSIGN () msg <$> defaultSignKeyGen
 
+defaultTestEnough :: QuickCheckTests -> QuickCheckTests
```

Comment:
Consider adding a comment that explains why this is necessary, and why 10,000 has been chosen as the limit.

Reply:
Thanks. Makes sense. Updated the comment.